spec/integration/systems_spec.rb

```diff
@@ -18,6 +18,7 @@
       auth_header
       pagination_params
       search_params
+      tags_params
```

Comment:
This should be added to all the specs because all our endpoints accept tags as implemented

Reply:
@akofink only the systems endpoint will allow tags, in every other case it'll ignore the param. So not sure if it should be everywhere :thinking: